core-configuration/src/main/config/properties/org/silverpeas/map/multilang/mapBundle_en.properties

```diff
@@ -22,6 +22,8 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 map.fullscreen.mode=Fullscreen mode
+map.target.point.none=No points have been targeted on the map
+map.target.point.title=Targeting a point on a map
```

Comment:
Target a point on a map

Reply:
Yes, it works too. (https://www.deepl.com/translator#fr/en/Cibler%20un%20point%20sur%20une%20carte)